docs/solution-architecture/accessControlPolicies.md

```diff
@@ -45,68 +45,68 @@ Finally, there is a special `domein_elk_zaaktype` system role which will grant t
+|                                               |                                                                                      |             |               |               |                                                                  |
+| **_Taak rechten_**                            |                                                                                      |             |               |               |                                                                  |
+| lezen                                         |                                          X                                           |             |       X       |               | OPA policy `taak lezen`                                          |
+| wijzigen                                      |                                          X                                           |             |       X       |               | OPA policy `taak wijzigen`                                       |
+| toekennen                                     |                                          X                                           |             |               |               | OPA policy `taak toekennen`                                      |
+| creeeren_document                             |                                    X (zaak open)                                     |             |       X       |               | OPA policy `zaak wijzigen`                                       |
+| toevoegen_document                            |                               X (zaak open, taak open)                               |             | X (taak open) |               | OPA policy `taak toevoegen_document`                             |
+|                                               |                                                                                      |             |               |               |                                                                  |
+| **_Document rechten_**                        |                                                                                      |             |               |               |                                                                  |
+| lezen                                         |                                          X                                           |      X      |       X       |               | OPA policy `document lezen`                                      |
+| wijzigen                                      | X (zaak open, document onvergrendeld, niet definitief of vergrendeld door gebruiker) |             |       X       |               | OPA policy `document wijzigen`                                   |
```

Comment:
Shame that this makes the table column even more wide because it is already so hard to read.. I don't know how to avoid it however without having to resort to HTML in the columns which we preferably do not want. Maybe it is unavoidable..

Reply:
It's hard to read on this window size yes
In a IDE it looks fine 
![afbeelding](https://github.com/infonl/dimpact-zaakafhandelcomponent/assets/15870935/ccac8a98-f529-451c-83e2-ce369b3ea94e)

what makes it a bit easier to read is using a rich diff in github, you can find tha toption here while reviewing
![afbeelding](https://github.com/infonl/dimpact-zaakafhandelcomponent/assets/15870935/a3726e9f-8a05-444c-b0ec-af1f8ab4d9f4)

Reply:
People from municipalities are reading this document and table on GitHub.

Reply:
I think we need to switch to something like this:

```X <ul><li>zaak open</li><li>niet heropend</li><li>niet opgeschort</li></ul>```

That's better readable in the end I think. But let's leave that for another PR I guess.